graphql/src/main/java/io/micronaut/configuration/graphql/GraphQLConfiguration.java

```diff
@@ -51,7 +51,7 @@ public class GraphQLConfiguration implements Toggleable {
     /**
      * The configuration name of the GraphQL path.
      */
-    public static final String PATH = PREFIX + ".path";
+    public static final String PATH_CONFIG = PREFIX + ".path";
```

Comment:
why do we need to change this? If we don't need it, lets change it to avoid unnecessary breaking changes.